webrtc/coverage/set-session-description.txt

```diff
@@ -0,0 +1,242 @@
+
+            [Untestable]
+            2.  Set channel's readyState attribute to closed.
+
+            [Untestable]
+            3.  Fire an event named error with a ResourceInUse exception at channel.
+
+            [Untestable]
+            4.  Fire a simple event named close at channel.
+
+    [TODO]
+    7.  If description is set as a local description, then run the following steps for
+        each media description in description that is not yet associated with an
+        RTCRtpTransceiver object:
+
+      [TODO]
+      1.  Let transceiver be the RTCRtpTransceiver used to create the media
+          description.
+
+      [TODO]
+      2.  Set transceiver's mid value to the mid of the corresponding media
+          description.
+
+    [TODO]
+    8.  If description is set as a remote description, then run the following steps
+        for each media description in description:
+
+      [TODO]
+      1.  As described by [JSEP] (section 5.9.), attempt to find an existing
+          RTCRtpTransceiver object, transceiver, to represent the media description.
+
+      [TODO]
+      2.  If no suitable transceiver is found (transceiver is unset), run the following
+          steps:
+
+        [TODO]
+        1.  Create an RTCRtpSender, sender, from the media description.
+
+        [TODO]
+        2.  Create an RTCRtpReceiver, receiver, from the media description.
+
+        [TODO]
+        3.  Create an RTCRtpTransceiver with sender, receiver and direction, and let
+            transceiver be the result.
+
+      [TODO]
+      3.  Set transceiver's mid value to the mid of the corresponding media description.
+          If the media description has no MID, and transceiver's mid is unset, generate
+          a random value as described in [JSEP] (section 5.9.).
+
+      [TODO]
+      4.  If the direction of the media description is sendrecv or sendonly, and
+          transceiver.receiver.track has not yet been fired in a track event, process
+          the remote track for the media description, given transceiver.
+
+      [TODO]
+      5.  If the media description is rejected, and transceiver is not already stopped,
+          stop the RTCRtpTransceiver transceiver.
+
+
+    [TODO]
+    9.  If description is of type "rollback", then run the following steps:
+
+      [TODO]
+      1.  If the mid value of an RTCRtpTransceiver was set to a non-null value by
+          the RTCSessionDescription that is being rolled back, set the mid value
+          of that transceiver to null, as described by [JSEP] (section 4.1.8.2.).
+
+      [TODO]
+      2.  If an RTCRtpTransceiver was created by applying the RTCSessionDescription
+          that is being rolled back, and a track has not been attached to it via
+          addTrack, remove that transceiver from connection's set of transceivers,
+          as described by [JSEP] (section 4.1.8.2.).
+
+      [TODO]
+      3.  Restore the value of connection's [[SctpTransport]] internal slot to its
```

Comment:
Yeah I am kind of loose between the terms "not tested", "untested", and "todo", though they all mean the same. "untested" is kind of look too similar to "untestable", "not tested" is consist of two words, and "todo" is kind of short more obvious on what still need to be tested.